workflow/snakemake/Snakefile

```diff
@@ -0,0 +1,45 @@
+# Note that if you are working on the analysis development locally, i.e. outside
+# of the REANA platform, you can proceed as follows:
+#
+#   $ mkdir snakemake-local-run
+#   $ cd snakemake-local-run
+#   $ virtualenv ~/.virtualenvs/root6-roofit-snakemake
+#   $ source ~/.virtualenvs/root6-roofit-snakemake/bin/activate
+#   $ pip install snakemake
+#   $ cp -a ../code ../workflow/snakemake/inputs.yaml .
```

Comment:
We can remove the `../workflow/snakemake/inputs.yaml` part as we will point to the path via `--configfile`.